# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/cache/Context.cpp
# Commit: 48eaa84c206d
# Full Hash: 48eaa84c206dbdda637c57b55ddbd07e975ba103
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2020-06-23 16:15:06
# Regressor Bug: 1637500
# File Overlap Count: 2
# Description:
#   Bug 1637500 - P2. Rename methods as they are not always dealing with "threads". r=froydnj
#   
#   Before P1, GetCurrentThreadSerialEventTarget would have always returned the same data as NS_GetCurrentThread, making the comment incorrect Now it will properly return the running TaskQueue if any.
#   
#   This change of name more clearly exposes what they are doing, as we aren't always dealing with threads directly; but a nsISerialEventTarget
# ==============================================================================

diff -r 00687d06a8a6 -r 48eaa84c206d dom/cache/Context.cpp
--- a/dom/cache/Context.cpp	Tue Jun 23 05:02:39 2020 +0000
+++ b/dom/cache/Context.cpp	Tue Jun 23 05:05:36 2020 +0000
@@ -97,7 +97,7 @@
         mData(aData),
         mTarget(aTarget),
         mInitAction(aInitAction),
-        mInitiatingEventTarget(GetCurrentThreadEventTarget()),
+        mInitiatingEventTarget(GetCurrentEventTarget()),
         mResult(NS_OK),
         mState(STATE_INIT),
         mCanceled(false) {
@@ -469,7 +469,7 @@
         mTarget(aTarget),
         mAction(aAction),
         mQuotaInfo(aQuotaInfo),
-        mInitiatingThread(GetCurrentThreadEventTarget()),
+        mInitiatingThread(GetCurrentEventTarget()),
         mState(STATE_INIT),
         mResult(NS_OK),
         mExecutingRunOnTarget(false) {
@@ -700,7 +700,7 @@
 Context::ThreadsafeHandle::ThreadsafeHandle(SafeRefPtr<Context> aContext)
     : mStrongRef(std::move(aContext)),
       mWeakRef(mStrongRef.unsafeGetRawPtr()),
-      mOwningEventTarget(GetCurrentThreadSerialEventTarget()) {}
+      mOwningEventTarget(GetCurrentSerialEventTarget()) {}
 
 Context::ThreadsafeHandle::~ThreadsafeHandle() {
   // Normally we only touch mStrongRef on the owning thread.  This is safe,